### DISCUSSION TITLE
NeoCodeium Chat

### DISCUSSION THREAD

**Discussion by gaven140** [monkoose/neocodeium#32] [Q&A]
`:NeoCodeium chat` is not opening chat tab/browser. it prints out "chat has been opened in the browser" but nothing is open in the browser. I am using kdeneon + qtile. default browser is firefox. Any body facing the same issue???

**Comment by monkoose** [comment]
What is the output of `xdg-settings get default-web-browser`? If it is not `firefox.desktop` then set it as default with `xdg-settings set default-web-browser firefox.desktop`
